08-transport.md

```diff
@@ -609,7 +609,243 @@ It is strongly recommended that existing, commonly-used, validated
 libraries be used for encryption and decryption, to avoid the many
 implementation pitfalls possible.
 
-## Acknowledgements
+# Appendix A: Transport Test Vectors
+
+To make a repeatable handshake, we specify what `generateKey()` will
+return (ie. the value for `e.priv`) for each side.  Note that this
+is a violation of the spec, which requires randomness here.
+
+## Initiator Tests
+
+The initiator should produce the given output when fed this input.
+The comments reflect internal state for debugging.
+
+    name: transport-initiator successful handshake
+    rs.pub: 0x028d7500dd4c12685d1f568b4c2b5048e8534b873319f3a8daa612b469132ec7f7
+    ls.priv: 0x1111111111111111111111111111111111111111111111111111111111111111
+    ls.pub: 0x034f355bdcb7cc0af728ef3cceb9615d90684bb5b2ca5f859ab0f0b704075871aa
+    e.priv: 0x1212121212121212121212121212121212121212121212121212121212121212
+    e.pub: 0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7
+    # Act One
+    # e.pub=0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7 e.priv=0x1212121212121212121212121212121212121212121212121212121212121212
+    # h=0x9e0e7de8bb75554f21db034633de04be41a2b8a18da7a319a03c803bf02b396c
```

Comment:
I think we should also specify what the starting `h` and `ck` values should be (for both sides before `ActOne`): 
```
h = 0x8401b3fdcaaa710b5405400536a3d5fd7792fe8e7fe29cd8b687216fe323ecbd
ck = 2640f52eebcd9e882958951c794250eedb28002c05d7dc2ea0f195406042caf1
```

This'll help implementers to ensure they initialized the state properly.